src/main/kotlin/com/saagie/technologies/SaagieTechnologiesPackageGradlePlugin.kt

```diff
@@ -113,15 +115,25 @@ class SaagieTechnologiesPackageGradlePlugin : Plugin<Project> {
                     }
                 }
                 metadata.contexts.forEach { context ->
-                    if (context.dockerInfo?.dynamicVersion != null &&
-                            context.dockerInfo.dynamicVersion.endsWith(dockerFormattedVersion)
-                    ) {
-                        logger.info("$it => ${context.dockerInfo.dynamicVersion}")
-                        promoteDockerImage(context.dockerInfo)
+                    run {
+                        fun checkVersionAndPromote(dockerInfo: DockerInfo?) {
+                            if (dockerInfo?.version != null && dockerInfo.version.endsWith(dockerFormattedVersion)) {
+                                logger.info("\t\t${it.toString()
+                                        .replace("technologies/job/", "")
+                                        .replace("/metadata.yaml", "")} => ${dockerInfo.version}")
+                                promoteDockerImage(dockerInfo)
+                            }
+                        }
+                        checkVersionAndPromote(context.dockerInfo)
+                        context.innerContexts?.forEach { innerContext ->
+                            innerContext.innerContexts?.forEach { finalContext ->
+                                checkVersionAndPromote(finalContext.dockerInfo)
+                            }
+                        }
                     }
                 }
                 tempFile.copyTo(file, true)
-                logger.info("${file.path} UPDATED")
```

Comment:
voluntarily change of log's level?

Reply:
Yes, this logs says updated no mater if there is a real update or if nothing changed, seems more like a debug log to me.
If changed should display "treated" for instance and maybe we could add another log telling when a real update was a done.